src/rabbit_amqqueue.erl

```diff
@@ -609,6 +621,21 @@ consumers_all(VHostPath) ->
                   {ChPid, CTag, AckRequired, Prefetch, Args} <- consumers(Q)]
           end)).
 
+consumers_all(VHostPath, Ref, Pid) ->
```

Comment:
I'd rename `Pid` to something that reflects that the pid/process is a result aggregator. `AggregatorPid`, maybe?